script/generate_views

```diff
@@ -1,195 +1,5 @@
-#!/usr/bin/env python3
+#!/bin/sh
 
-"""
-Generate one view definition file per document type in '_stable' tables.
+cd "$(dirname "$0")/.."
 
-If there are existing view definitions in the destination directory then those will be
-kept instead.
-
-Run as:
-  ./script/generate_views 'moz-fx-data-shared-prod:*_stable.*'
```

Comment:
It would be useful to include a comment block at the top that preserves a basic explanation of what the script does and how to invoke it.